### PR TITLE
Update all Option::None(()) to remove (())

### PR DESCRIPTION
**Pull request by gaetbout**
Updated:
 - break (); => break;
 - Option::None(()) => Option::None
 - Option::None(_) => Option::None

### CODE REVIEW

File: src/data_structures/tests/stack_test.cairo [hunk 1]
```diff
@@ -44,7 +44,7 @@ fn stack_peek_test<
         Option::Some(result) => {
             assert(result == val_2, 'wrong result');
         },
-        Option::None(_) => {
+        Option::None => {
```
Comment (LucasLvy) [inline src/data_structures/tests/stack_test.cairo:47]: `assert(false, blablabla..);`

**Review by LucasLvy — CHANGES_REQUESTED** [review]
you get the idea

File: src/data_structures/stack.cairo [hunk 2]
```diff
@@ -74,7 +74,7 @@ impl Felt252StackImpl<
     /// * Option<u256> The item removed or None if the stack is empty.
     fn pop(ref self: Felt252Stack<T>) -> Option<T> {
         if self.is_empty() {
-            return Option::None(());
+            return Option::None;
```
Comment (LucasLvy) [inline src/data_structures/stack.cairo:77]:
```suggestion
            Option::None;
```

**Review by LucasLvy — CHANGES_REQUESTED** [review]
c'est reparti pour un tour

File: src/data_structures/stack.cairo (hunk 2 above)
Comment (gaetbout) [reply src/data_structures/stack.cairo:77]: image

Comment (LucasLvy) [reply src/data_structures/stack.cairo:77]: indeed, didn't see the end of the function

**Pull request merged**